main.cpp

```diff
@@ -583,7 +556,7 @@ float TempoEnv::newPoint(float newPointPosX, float newPointPosY,bool pointShape)
 
 void TempoEnv::deletePoint(int pointIndex)
 {
-    int newIndex = 0;
+    int newIndex;
     newIndex = pointIndex * 1;
```

Comment:
`int newIndex = pointIndex * 1;`
there is no need for the uninitialized variable on line 559, followed by assigning to it on the next line.
Just do it all in a single line.
- [ ] revise